app/src/main/java/org/mozilla/reference/browser/Components.kt

```diff
@@ -133,4 +140,32 @@ class Components(
             init()
         }
     }
+
+    private val sentryService: SentryService by lazy {
+        SentryService(
+            applicationContext,
+            SENTRY_TOKEN,
+            sendEventForNativeCrashes = true
+        )
+    }
+    private val socorroService: MozillaSocorroService by lazy {
+        MozillaSocorroService(applicationContext, "ReferenceBrowser")
+    }
+
+    val crashReporter: CrashReporter by lazy {
+        CrashReporter(
+            services = listOf(sentryService, socorroService),
```

Comment:
nit: I wonder if we should just create the services in here. No one else properly needs access to them. :)